plugins/cache/redis.py

```diff
@@ -32,6 +32,14 @@
         ini:
           - key: fact_caching_prefix
             section: defaults
+      _cache_keys_name:
+        description: User defined name for ansible cache key name
+        default: ansible_cache_keys
+        env:
+          - name: ANSIBLE_CACHE_KEYS_NAME
```

Comment:
Won't it make sense to include `REDIS` in the key name? (Also for the `ini` key.)

Reply:
I checked the other available plugins (yaml, memcached etc.). they share the same configuration (_uri, _prefix, _timeout).
This change7add is exclusive to this cache module and therefore I agree with you, as its redis specific.

Reply:
I renamed it to `keyset_name` as it's used als as variable in the code.